Horizon/Horizon/Sources/Features/HorizonTabBarController.swift

```diff
@@ -68,4 +69,22 @@ class HorizonTabBarController: UITabBarController, UITabBarControllerDelegate {
         vc.tabBarItem.image = UIImage(systemName: "newspaper")
         return vc
     }
+
+    private func inboxTab() -> UIViewController {
+        let inboxController: UIViewController
+        let inboxSplit = CoreSplitViewController()
+
+        inboxController = InboxAssembly.makeInboxViewController()
+
+        let empty = CoreNavigationController()
+
+        inboxSplit.viewControllers = [inboxController, empty]
+        let title = "Inbox"
+        inboxSplit.tabBarItem = UITabBarItem(title: title, image: .inboxTab, selectedImage: .inboxTabActive)
+        inboxSplit.tabBarItem.accessibilityIdentifier = "TabBar.inboxTab"
+        inboxSplit.tabBarItem.makeUnavailableInOfflineMode()
```

Comment:
Please comment out this for now as we don't have exact plans for offline support.